projects/Mallard/src/helpers/files.ts

```diff
@@ -185,13 +186,21 @@ export const downloadAndUnzipIssue = async (
 }
 
 export const clearOldIssues = async () => {
-    const edition = 'daily-edition'
-    const files = await RNFetchBlob.fs.ls(`${FSPaths.issuesDir}/${edition}`)
+    const files = await RNFetchBlob.fs.ls(
+        `${FSPaths.issuesDir}/${defaultSettings.contentPrefix}`,
+    )
 
     const issuesToDelete = files.filter(
-        issue => !lastSevenDays().includes(issue),
+        issue => !lastSevenDays().includes(issue) && issue === 'issues',
     )
-    issuesToDelete.map(issue => RNFetchBlob.fs.unlink(FSPaths.issueRoot(issue)))
+
+    issuesToDelete.map(issue => {
+        RNFetchBlob.fs
+            .unlink(
+                `${FSPaths.issuesDir}/${defaultSettings.contentPrefix}/${issue}`,
```

Comment:
Same here?

Reply:
Sure - shall we get this merged and i can look at a better strategy for centralising these being used and write some tests in another PR?